Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -698,16 +698,45 @@ static GridArea insertIntoGrid(Grid& grid, RenderBox& child, const GridArea& are
     return clamped;
 }
 
-static inline bool isMasonryRows(const RenderStyle& style)
+bool RenderGrid::isMasonryRows() const
 {
-    return style.gridMasonryRows();
+    return style().gridMasonryRows();
 }
 
-static inline bool isMasonryColumns(const RenderStyle& style)
+// Masonry Spec Section 2
+// "If masonry is specified for both grid-template-columns and grid-template-rows, then the used value for grid-template-columns is none,
+// and thus the inline axis will be the grid axis."
+
```

Comment:
No whitespace between method comment and method definition, please.

Reply:
Fixed